charts/seed-controlplane/charts/kube-controller-manager/templates/kube-controller-manager-networkpolicy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: {{ include "networkpolicyversion" . }}
+kind: NetworkPolicy
+metadata:
+  name: kube-controller-manager-allow-metadata-service
+  namespace:  {{ .Release.Namespace }}
+spec:
+  podSelector:
+    app: kubernetes
```

Comment:
Validation Error: It should be 
```yaml
podSelector:
  matchLabels:
    app: kubernetes
```

Reply:
I forgot to stage that change when I did the commit